src/components/map-page-components/mobile-landing-bar/MobileLandingBar.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
```

Comment:
I probably should have uh written this down somewhere, or changed frontend scaffold, idk, but we should be using `lowerCamelCase` for directory names. 

You can also name the component `index.tsx` and import it without specifying the full file path. 

No changes required on these, but if you want to :)

Reply:
I can open a PR to fix all the other directory names too.

Reply:
That would be excellent, cc @florisdobber would be good to track this in monday